### PR TITLE
fix for issue 99

### PR DESCRIPTION
**Pull request by ajay74984**

### CODE REVIEW

**Comment by ajay74984** [comment]
@magwas  ready for review.

File: src/main/java/com/kodekonveyor/work_request/find/FindWorkRequestController.java [hunk 1]
```diff
@@ -27,16 +31,25 @@
...
       final String country, final String city,
       final String workType
   ) {
+    loggerService.info(WorkRequestConstants.WORK_REQUEST_FIND);
     WorkTypeValidationUtil.validateWorkType(workType);
     CountryAndCityValidationUtil.validateCountry(country);
 
     CountryAndCityValidationUtil.validateCity(city);
 
+    loggerService.debug(
```
Comment (magwas) [inline src/main/java/com/kodekonveyor/work_request/find/FindWorkRequestController.java:48]: you don't need to log this step

File: src/test/java/com/kodekonveyor/work_request/find/FindWorkRequestControllerTestBase.java [hunk 2]
```diff
@@ -19,6 +20,9 @@
   @Mock
   AuthenticatedUserService authenticatedUserService;
 
+  @Mock
+  Logger loggerService; //NOPMD
```
Comment (magwas) [inline src/test/java/com/kodekonveyor/work_request/find/FindWorkRequestControllerTestBase.java:24]: why this NOPMD here?

Comment (ajay74984) [reply src/test/java/com/kodekonveyor/work_request/find/FindWorkRequestControllerTestBase.java:24]: got a reference from some other tests and it was present there. removed in latest commit though.

File: src/main/java/com/kodekonveyor/work_request/find/FindWorkRequestController.java (hunk 1 above)
Comment (ajay74984) [reply src/main/java/com/kodekonveyor/work_request/find/FindWorkRequestController.java]: removed.

**Comment by ajay74984** [comment]
@magwas  its ready to be merged.

**Comment by ajay74984** [comment]
@magwas Please review the changes.

**Comment by magwas** [comment]
please solve the conflict

**Comment by magwas** [comment]
closing due to inactivity. pleae regrab if you want to continue

**Pull request closed without merging**